app/models/ext_management_system.rb

```diff
@@ -27,6 +27,10 @@ def self.supported_types_and_descriptions_hash
     end
   end
 
+  def self.api_allowed_attributes
+    %w()
```

Comment:
Ah I see you've changed it to a method. You'll need to update the commit message, as it still mentions it being a constant.

Reply:
@juliancheal updated ✨